packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -98,16 +98,18 @@ describe('Select', () => {
         async function setup500Options(): Promise<Fixture<Select>> {
             // prettier-ignore
             const viewTemplate = html`
-                <nimble-select>
+                <nimble-select position="below">
                     ${repeat(() => [...Array(500).keys()], html<number>`
                         <nimble-list-option value="${x => x}">${x => x}</nimble-list-option>`)}
                 </nimble-select>
             `;
             return fixture<Select>(viewTemplate);
         }
 
-        // Disabled due to intermittancy, see: https://github.com/ni/nimble/issues/1172
-        xit('should limit dropdown height to viewport', async () => {
+        // The select must be set to position the list below, because the interaction
+        // between the karma tester's grid of dots, starting a new dot row, and automatic
+        // scrolling can cause this test to fail when the position is above.
+        it('should limit dropdown height to viewport', async () => {
```

Comment:
I confirmed that works. It's essentially doing the same thing by causing the select to always open to the bottom. But as you say, this could prevent issues with other tests, and I can't see any harm. I've adopted your approach for this PR.